plaidml/bridge/openvino/tests/functional/plugin/plaidml/shared_tests_instances/single_layer_tests/squared_difference.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <cstddef>
+#include <map>
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/squared_difference.hpp"
+
+using LayerTestsDefinitions::SquaredDifferenceLayerTest;
+
+namespace {
+const std::vector<InferenceEngine::Precision> inputPrecisions = {
+    InferenceEngine::Precision::FP32,
+};
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+};
```

Comment:
Only need one precision for the style of test we're using (I think I've mostly been calling it netPrecision, although with this style of test they're the same thing)